.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,92 @@
+name: Bug Report
+description: Report a bug
+#title: ""
+labels: ["bug"]
+#assignees:
+#  -
+body:
+  - type: checkboxes
+    attributes:
+      label: Is there an existing issue for this?
+      description: Please search to see if an issue already exists for the bug you encountered.
+      options:
+      - label: I have searched the existing issues
+        required: true
+  - type: textarea
+    id: summary
+    attributes:
+      label: Summary
+      description: A clear and specific description of what the bug is.
+    validations:
+      required: true
+  - type: textarea
+    id: steps-to-reproduce
+    attributes:
+      label: Steps to reproduce the behavior
+      placeholder: |
+        1. Go to '...'
+        2. Click on '...'
+        3. Scroll down to '...'
+    validations:
+      required: true
+  - type: textarea
+    id: actual-result
+    attributes:
+      label: Actual Result
+      description: What is the reproducible outcome?
+      placeholder: See error...
+    validations:
+      required: true
+  - type: textarea
+    id: expected-result
+    attributes:
+      label: Expected Result
+      description: A clear and concise description of what you expected to happen.
+      placeholder: No errors observed.
+    validations:
+      required: true
+  - type: textarea
+    id: what-have-you-tried
+    attributes:
+      label: What have you tried to resolve this issue?
+      description: What steps have you taken to try and understand the issue? Context will enable us to help you faster, or, even better, will help you solve it yourself!
+    validations:
+      required: true
+  - type: checkboxes
```

Comment:
Should all these "have you checked X" things be at the top in one thing? There could be a block of three (if we want them)